src/util/fields.py

```diff
@@ -731,6 +733,83 @@ def convert_field_to_html(cr, model, field, skip_inherit=()):
         convert_field_to_html(cr, inh.model, field, skip_inherit=skip_inherit)
 
 
+def _get_default_m2m_table_name(table1, table2):
+    return "{}_{}_rel".format(*sorted([table1, table2]))
+
+
+def convert_many2one_field_to_many2many(cr, model, field, new_name=None, m2m_table=None, col1=None, col2=None):
```

Comment:
```suggestion
def convert_m2o_field_to_m2m(cr, model, field, new_name=None, m2m_table=None, col1=None, col2=None):
```

In all other utils (create_m2m, fixup_m2m and a dozen more) we use the shorthand form (`m2{m,o}` instead of `many2{many,one}`, shouldn't we keep it consistent here as well?

Reply:
You're right.